debugtools/DDR_VM/src/com/ibm/j9ddr/tools/ddrinteractive/commands/ForeachCommand.java

```diff
@@ -60,7 +60,7 @@ public void run(String command, final String[] args, final Context context, fina
 					out.println();
 					out.println("\techo\t\tOutput each command before it is run.");
 					out.println("\tquiet\t\tSuppress delimiters between command outputs.");
-					out.println("\ttoken=<chars>\tSpecify a substitution token. Occurrencs of this string in the args will be replaced by the current element.");
+					out.println("\ttoken=<chars>\tSpecify a substitution token. Occurrences of this string in the args will be replaced by the current element.");
```

Comment:
This isn't a comment.

Reply:
Will drop

Reply:
Isn't this an improvement still, irregardless of the fact that it's not in a comment. I would vote to keep this valid change.

Reply:
I'll get it in another pass.

Reply:
We agreed to do non-comment changes separately because they require more testing.

Reply:
I see, I'm ok with that. Thanks!